python/tests/compat/arcticdb/test_compatibility.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
+import pandas as pd
+import pytest
+from arcticdb.arctic import Arctic
+from arcticdb.util.test import assert_frame_equal
+from arcticdb_ext import set_config_int
+from arcticdb.options import ModifiableEnterpriseLibraryOption
+
+def test_compat_write_read(old_venv, arctic_uri):
```

Comment:
I think we discussed having 2 Arctic venvs as fixtures to avoid any dependency on the current version?

Reply:
We discussed having different Arctic venvs as different fixtures if for some reason we need to test some very specific issue where e.g. 3 different versions are involved. We can easily separate those out of the `old_venv` fixture if needed in the future but for sake of simplicity I'm keeping it as is as now we don't need it.

Dependening on current version is critical I think. E.g. if we just have a test running only 1.6.2 and 4.5.0 we will never catch a regression in the current version which is what these tests are for. We want to know if a change we make will break e.g. modifying a 1.6.2 library in some weird surprising way.